### PR TITLE
RECIRC-159: Include FANDOM_TOPIC items in IMPACT_FOOTER scroller

### PR DESCRIPTION
**Pull request by pauloslund**
This change also starts the process of moving placement experiments into their own files in hopes of cleaning up the `placement.js` and making new groups easier to add

### CODE REVIEW

File: extensions/wikia/Recirculation/RecirculationHooks.class.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public static function onOasisSkinAssetGroups( &$jsAssets ) {
 		if ( self::isCorrectPageType() ) {
 			$jsAssets[] = 'recirculation_js';
 
-			if ( self::canShowDiscussions() ) {
+			if ( self::canShowDiscussions( $wgCityId ) ) {
```
Comment (nmonterroso) [inline extensions/wikia/Recirculation/RecirculationHooks.class.php:59]: seems like this check is being done many times - is there a way to consolidate this or are the use cases all different?

**Comment by nmonterroso** [comment]
small comment otherwise lgtm

File: extensions/wikia/Recirculation/RecirculationHooks.class.php (hunk 1 above)
Comment (pauloslund) [inline extensions/wikia/Recirculation/RecirculationHooks.class.php:59]: The use cases are a little bit different as they are for different experiments, however the check could probably be removed from the `RecirculationController.php` file as the route that uses it should never get called without including the `recirculation_discussions_js` file referenced here. The Recirculation controller is getting an update in the near future and will probably be deprecated in favor of just using the RecirculationAPIController so I'll take a look at consolidating it then.

Comment (nmonterroso) [inline extensions/wikia/Recirculation/RecirculationHooks.class.php:59]: cool, LGTM then 👍

**Pull request merged**